assets/app/views/_triggers.html

```diff
@@ -3,9 +3,16 @@
     <span ng-switch="build.status" class="hide-ng-leave">
       <span ng-switch-when="Complete" class="fa fa-check text-success" aria-hidden="true" style="margin-right: 5px;"></span>
       <span ng-switch-when="Failed" class="fa fa-times text-danger" aria-hidden="true" style="margin-right: 5px;"></span>
+      <span ng-switch-when="Cancelled" class="fa fa-ban text-warning" aria-hidden="true"></span>
```

Comment:
do we know all the states? Seems like we might want a hidden icon to be the default, and only show spinning for "Pending", "Running", etc